MachineLearningPrototype/collect_data.py

```diff
@@ -0,0 +1,132 @@
+import requests
+import argparse
+from datetime import datetime
+from datetime import date
+from math import ceil
+import time
+from pprint import pprint
+
+def get_num_pages(repo):
+	"""
+	Fetch the number of pages that store issues
+	"""
+	repo_query_url = f'https://api.github.com/repos/{repo}'
+	num_issues = requests.get(repo_query_url).json()['open_issues_count']
+	num_pages = ceil(num_issues/100)
+	return num_pages
+
+def fetch_all_new_issues(repo, since, auth_token, labels):
```

Comment:
I just created that function so that we can use it whenever we want to fetch all the issues of a repo. The `fetch_issues_with_labels` function is used to fetch issues with a given label.